bin/git-make-release

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# Prompt for the version number
+prompt_for_version() {
+  read -rp "Enter the version number (default: $1): " version
+  echo "${version:-$1}" # Return the entered version number or the default if none was entered
```

Comment:
Should we have a default value here? Seems like it is more likely to lead to incorrect outcomes than correct outcomes if applied

Reply:
Discussed in meet, my misunderstanding